knightos/commands/init.py

```diff
@@ -66,8 +66,18 @@ def init(ws, root, exists, site_packages, template, template_vars, vcs, force):
         install_local_kernel(ws.root, ws.kroot, template_vars['platform'], template_vars['kernel_path']);
 
     print("Installing packages...")
-    install.execute(template["install"])
     ws = Workspace(root)
+
+    filtered_packages = template["install"]
+    to_exclude = ws.config.get("-exclude-template-deps")
+    if ws.config.get("-exclude-template-deps"):
+        to_exclude = to_exclude.split(" ")
+        for package in to_exclude:
+            if package in filtered_packages:
+                filtered_packages.remove(package)
+                print("Not installing package " + package + " as specified by -exclude-template-deps")
```

Comment:
This should only be done in verbose mode (if such a thing exists). If we don't have a verbose mode, I'd just comment it out.